check_if_image_tag_exists/test_config.json

```diff
@@ -1,4 +1,5 @@
 {
+	"schemaVersion": "1.0.0",
 	"commandTests": [
```

Comment:
Is this just for testing purposes? It would be good to try to keep everything at just one schema for as long as possible.